cnv/cnv_install/cnv-about-cnv.adoc

```diff
@@ -0,0 +1,35 @@
+[id="cnv-about-cnv"]
+= About {ProductName}
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-about-cnv
+toc::[]
+
+{ProductName} is an add-on to {product-title} that allows you to run and manage
+ virtual machine workloads alongside container workloads. 
+
+{ProductName} adds new objects into your {product-title} cluster via Kubernetes custom resources that enable virtualization tasks, including:
+
+* Creating and managing virtual machines 
+* Supported drivers for Windows virtual machines
+* Importing and cloning existing virtual machines, including VMware virtual machines
+* Additional network interface controllers and storage disks to virtual machines
+* Live migration of virtual machines between nodes
+
+An enhanced web console provides a graphical portal to manage these virtualized 
+resources alongside the {product-title} cluster containers and infrastructure.
+
+[IMPORTANT]
```

Comment:
Please reuse the tech preview module in the assembly instead.
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features